workbench_access.post_update.php

```diff
@@ -96,3 +97,61 @@ function workbench_access_post_update_convert_user_storage_keys(array &$sandbox)
   $sandbox['#finished'] = empty($sandbox['ids']) ? 1 : ($sandbox['count'] - count($sandbox['ids'])) / $sandbox['count'];
   return t('Updated user assigments');
 }
+
+/**
+ * Transform existing role data to new storage.
+ */
+function workbench_access_post_update_section_role_association(&$sandbox) {
+  $schemes = \Drupal::entityTypeManager()->getStorage('access_scheme')->loadMultiple();
+  $storage = \Drupal::service('workbench_access.role_section_storage');
+  $state = \Drupal::state();
+  foreach ($schemes as $scheme) {
+    foreach (\Drupal::entityTypeManager()->getStorage('user_role')->loadMultiple() as $rid => $role) {
+      $potential_ids = [
+        RoleSectionStorageInterface::WORKBENCH_ACCESS_ROLES_STATE_PREFIX . $rid,
```

Comment:
If I understand the post_update hook correctly, we do not need this item, as it has already been converted to __default.